services/121-service/test/helpers/registration.helper.ts

```diff
@@ -17,7 +17,7 @@ export function importRegistrations(
   accessToken: string,
 ): Promise<request.Response> {
   return getServer()
-    .post(`/programs/${programId}/registrations/import`)
+    .post(`/programs/${programId}/registrations`)
```

Comment:
you forgot to make this same change in seed-mock-helper > importRegistrations(). Changing that fixes the the mock seed script again.